src/printer/embed.ts

```diff
@@ -50,7 +58,7 @@ export function embed(
 			return doc;
 		});
 
-		return ['{', astroDoc, '}'];
+		return group(['{', indent([softline, astroDoc]), softline, lineSuffixBoundary, '}']);
```

Comment:
This matches Prettier's behaviour inside JSX files much better than what we had before. Previously, we'd never put the content on another line nor put a newline between the content and the ending curly bracket, which wasn't too pretty in certain cases

Reply:
Nice!